nleis/fitting.py

```diff
@@ -197,193 +187,7 @@ def set_default_bounds(circuit, constants={}):
 
     return bounds
 
```

Comment:
_circuit_fit_ and _wrapCircuit_ needs to be restored

Reply:
I think it might be simpler to just preprocess the bounds ahead of calling the circuit_fit function (my understanding is the the only reason you need to copy circuit_fit is for the updated default bounds function)

Reply:
Yes, that will resolve the bounding issues. But I think there are still dependency issues for the _curve_fit_ and _wrapCircuit_ functions. They all depend on the edited _buildCircuit_ function to parse the _d()_. Again the _curve_fit_ and _wrapCircuit functions use our customized function. Please take a look at the following error message for it. I think we should stick to what I have previously to ensure the proper functionality of the code.  
<img width="975" alt="image" src="https://github.com/user-attachments/assets/1aa0809b-6d91-46d3-aca4-b9a37a339931">

Reply:
Let me know if you might have any concerns, but I think this is the problem we have right now